pages/common/erl.md

```diff
@@ -0,0 +1,15 @@
+# erl
+
+> Run and manage programs written in Erlang programming language.
+
+- Run sequential erlang program as a common script and then exit:
+
+`erlc *.erl && erl -noshell '{{mymodule:myfunction(arguments)}}, init:stop().'`
+
+- Connect to a running erlang node:
+
+`erl -remsh {{nodename}}@{{hostname}} -sname {{custom_shortname}} -hidden -setcookie {{cookie_of_remote_node}}`
+
+- Tell the erlang shell to load another modules or other versions:
```

Comment:
Perhaps ....`another module or version:` would read better?

Reply:
Going to reword to "another modules or version". Is it ok?